src/components/common/styled-components.js

```diff
@@ -540,3 +553,33 @@ export const MapControlButton = styled.div.attrs({
     color: ${props => props.theme.secondaryBtnActColor};
   }
 `;
+
+export const StyledFilterPanel = styled.div`
+  margin-bottom: 12px;
+  border-radius: 1px;
+
+  .filter-panel__filter {
+    margin-top: 24px;
+  }
+`;
+
+export const StyledFilterHeader = styled(StyledPanelHeader)`
```

Comment:
this is only used in `filter-panel-header`, if it's not used in other places, I would keep it there